src/main/java/com/swp/user/controller/RelationshipController.java

```diff
@@ -0,0 +1,28 @@
+package com.swp.user.controller;
+
+import com.swp.user.domain.RelationshipService;
+import com.swp.user.dto.RelationshipRequestDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+public class RelationshipController {
+
+    private final RelationshipService relationshipService;
+
+    @PostMapping(value = "/v1/relationship")
+    public void followUser(@RequestBody RelationshipRequestDto relationshipRequestDto){
+        relationshipService.createRelationship(relationshipRequestDto);
```

Comment:
리소스가 생성됨을 표현하는 HTTP Status CREATED(201)을 주는 것은 어떨까요?
아래처럼 추가해주면 돼요!
```suggestion
    @PostMapping(value = "/v1/relationship")
    @ResponseStatus(HttpStatus.CREATED)
    public void followUser(@RequestBody RelationshipRequestDto relationshipRequestDto){
        relationshipService.createRelationship(relationshipRequestDto);
```

Reply:
넵